version.tf

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+terraform {
+  required_version = "~> 0.11.1"
+}
+
+provider "aws" {
```

Comment:
it is better use ~> to limit version between 1.60 and 2.00
provider "aws" {
  version = "~> 1.60"
}
https://www.terraform.io/docs/providers/aws/guides/version-2-upgrade.html

or using required_providers in terraform block
terraform {
  required_providers = {
    aws = "~> 1.60.0"
  }
}
https://www.terraform.io/docs/configuration/terraform.html

Reply:
`1.60.0` is the last release for `1.x` version so used this.
https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1600-february-22-2019

Reply:
updating PR to 
```
  required_providers = {
    aws = "~> 1.60.0"
 
```